src/sentry/incidents/subscription_processor.py

```diff
@@ -408,11 +309,33 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
                 },
             )
 
+        has_metric_alert_processing = features.has(
+            "organizations:workflow-engine-metric-alert-processing", organization
+        )
+        comparison_delta = None
+
+        if (
+            has_metric_alert_processing
+            and not self.alert_rule.detection_type == AlertRuleDetectionType.DYNAMIC
+        ):
+            try:
+                detector = Detector.objects.get(
+                    data_sources__source_id=str(self.subscription.id),
+                    data_sources__type=DATA_SOURCE_SNUBA_QUERY_SUBSCRIPTION,
+                )
+                comparison_delta = detector.config.get("comparison_delta")
+            except Detector.DoesNotExist:
```

Comment:
maybe do `sentry_sdk.capture_exception()` here so we can see the sentry error?

Reply:
This did capture the error in https://sentry.sentry.io/issues/6616374058?project=1